pkg/scheduler/framework/framework.go

```diff
@@ -373,7 +380,39 @@ func (f *framework) markAsUnscheduledFor(ctx context.Context, bindings []*placem
 					// We will just retry for conflict errors since the scheduler holds the truth here.
 					if apierrors.IsConflict(err) {
 						// get the binding again to make sure we have the latest version to update again.
-						return f.client.Get(cctx, client.ObjectKeyFromObject(unscheduledBinding), unscheduledBinding)
```

Comment:
Just returning error from GET call eats up error from the update call above. I'm not sure how we have not encountered issues till now